props/saber_BC_buttons.h

```diff
@@ -437,6 +429,53 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
     SaberBase::DoEffect(EFFECT_FAST_ON, 0);
   }
 
+  // Go to the next Preset skipping Preon effect with FastOn.
+  virtual void next_preset_fast() {
```

Comment:
@profezzorn I don't see this anywhere to remove it, so the "move to prop_base" is a move.
I click edit file and this section is not there.